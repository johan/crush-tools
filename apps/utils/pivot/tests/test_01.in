Buy-ID	Activity-Type	Activity-Sub-Type	Quantity	Revenue	Event-ID
	2005t352	homep274	1	0.00	0
	2005t352	homep274	1	0.00	0
	2005t352	homep274	1	0.00	0
	2005t352	homep274	1	0.00	0
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
	2005t352	homep274	1	0.00	0
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
	2005t352	conta474	1	0.00	0
	2005t352	conta474	1	0.00	0
	2005t352	conta474	1	0.00	0
	2005t352	homep274	1	0.00	0
	2005t352	homep274	1	0.00	0
	2005t352	homep274	1	0.00	0
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
	2005t352	homep274	1	0.00	0
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
	2005t352	homep274	1	0.00	0
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	1
	2005t352	homep274	1	0.00	0
1519200	2005t352	homep274	1	0.00	1
	2005t352	conta474	1	0.00	0
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	1
	2005t352	homep274	1	0.00	0
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
	2005t352	homep274	1	0.00	0
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
	2005t352	homep274	1	0.00	0
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	2
1519200	2005t352	homep274	1	0.00	2
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	conta474	1	0.00	1
1519200	2005t352	homep274	1	0.00	1
1519200	2005t352	homep274	1	0.00	2
