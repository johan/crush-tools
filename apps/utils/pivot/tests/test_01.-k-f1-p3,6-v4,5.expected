Buy-ID	conta474 - 0: Quantity	conta474 - 0: Revenue	conta474 - 1: Quantity	conta474 - 1: Revenue	conta474 - 2: Quantity	conta474 - 2: Revenue	homep274 - 0: Quantity	homep274 - 0: Revenue	homep274 - 1: Quantity	homep274 - 1: Revenue	homep274 - 2: Quantity	homep274 - 2: Revenue
	4	0.00	0	0.00	0	0.00	14	0.00	0	0.00	0	0.00
1519200	0	0.00	8	0.00	15	0.00	0	0.00	20	0.00	38	0.00
